### ISSUE TITLE
Update links on About page

### ISSUE THREAD

**Issue by tomsedory** [OpenEugene/little-help-book-web#171]
The [About](https://little-help-book.netlify.app/table-of-contents-style-homepage/about.html) page has three links with issues:

- For an improved user experience, the e-mail link should be a clickable link, not simply plain text.
     <img width="357" alt="Email link" src="https://user-images.githubusercontent.com/75692630/104547535-a474fe80-55e3-11eb-8533-ebcc8301ae67.png">

- The slack link should go to the proj-little-help-book channel, not the general channel in Open Eugene's workspace
     <img width="467" alt="Slack link" src="https://user-images.githubusercontent.com/75692630/104547548-accd3980-55e3-11eb-8187-21fcf6533ea5.png">

- The Sustainability link to open collective should pull up a page, not simply spin like it does now
     <img width="242" alt="Sustainability link" src="https://user-images.githubusercontent.com/75692630/104547564-b22a8400-55e3-11eb-839f-5fe90d73e622.png">
     <img width="998" alt="Sustainability page" src="https://user-images.githubusercontent.com/75692630/104547568-b48cde00-55e3-11eb-8fdc-cc2cb2c01a27.png">
